airqo-platform/AirQo-frontend
# [NetManager] KCCA dashboard feedback implementation

#### Summary of Changes (What does this PR do?)
- Fixed KCCA dashboard feedback

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* checkout this branch locally
* Install requirements `npm install`
* Start app `npm run stage-mac (macOS)` or `npm run stage-pc (windows Platform)`
* Verify that items on the "done" sub-tickets of [PLAT-362](https://airqoteam.atlassian.net/browse/PLAT-362).

#### What are the relevant tickets?
- [PLAT-362](https://airqoteam.atlassian.net/browse/PLAT-362)

#### Screenshots (optional)
